skins/wmde19/src/components/layout/Footer.vue

```diff
@@ -1,11 +1,81 @@
 <template>
-	<div class="footer">
-		Footer
+	<div class="footer has-background-footer">
+		<div class="container">
+			<div class="columns">
+				<div class="column is-two-thirds">
+					<div class="level">
+						<div class="level-left">
+							<div class="logo-footer">
+								<a href="https://www.wikimedia.de/">
+									<img :src="imagePath + '/images/logo-vertical-wikimedia.svg'" alt="Wikimedia Deutschland">
+								</a>
+							</div>
+						</div>
+						<div class="level-item">
+							<div class="bank-data-content">
+								<p><strong>Kontoinhaber:</strong> Wikimedia Foerdergesellschaft</p>
+								<p>
+									<strong>{{ $t( 'iban')  }}:</strong><span class="spacer" v-for="ibanPartial in iban">{{ ibanPartial }}</span>
+								</p>
+								<p><strong>{{ $t( 'bic')  }}:</strong> {{ $t( 'operator_account_bic')  }}</p>
+								<p>Bank für Sozialwirtschaft</p>
+								<p><strong>{{ $t( 'account_number')  }}:</strong> {{ $t( 'operator_account_number')  }}</p>
+								<p><strong>{{ $t( 'blz')  }}:</strong> {{ $t( 'operator_account_blz')  }}</p>
+							</div>
+						</div>
+					</div>
+				</div>
+				<div class="column is-one-third">
+					<ul class="list-menu">
+						<li v-for="link in footerMenu">
```

Comment:
`eslint` complains there's no `:key` passed for the different nodes.
https://vuejs.org/v2/guide/list.html#key
Also on the other places with `v-for`